src/Data/Managers/V2/GetCurrentUser/useGetCurrentUser.ts

```diff
@@ -0,0 +1,44 @@
+import { UseQueryResult, useQuery } from "@tanstack/react-query";
+import { PrimaryBaseUrlManager } from "@/UI";
+import { useFetchHelpers } from "../helpers";
+
+interface LoggedUser {
+  username: string;
+}
+/**
+ * Custom hook for getting the current logged in user from the server.
```

Comment:
Can we be clearer here, and specify those are React Queries?

Reply:
updated all hooks